python/semantic_kernel/connectors/ai/open_ai/services/open_ai_chat_completion_base.py

```diff
@@ -44,11 +46,16 @@
 from semantic_kernel.filters.filter_types import FilterTypes
 from semantic_kernel.filters.kernel_filters_extension import _rebuild_auto_function_invocation_context
 from semantic_kernel.functions.function_result import FunctionResult
+from semantic_kernel.utils import model_diagnostics
 
 if TYPE_CHECKING:
     from semantic_kernel.functions.kernel_arguments import KernelArguments
     from semantic_kernel.kernel import Kernel
 
+
+MODEL_PROVIDER_NAME = 'openai'
```

Comment:
let's move this into "semantic_kernel/connectors/ai/open_ai/const.py"

Reply:
or when combined with a decorator we should set them as ClassVar's in all completion classes.

Reply:
Gong for the ClassVar approach